src/Module.Server/Modes/Dtv/CrpgDtvServer.cs

```diff
@@ -307,11 +308,9 @@ private int GetDefendersCount()
         int defendersCount = 0;
         foreach (NetworkCommunicator networkPeer in GameNetwork.NetworkPeers)
         {
-            var missionPeer = networkPeer.GetComponent<MissionPeer>();
+            var crpgPeer = networkPeer.GetComponent<CrpgPeer>();
             if (!networkPeer.IsSynchronized
-                || missionPeer == null
-                || missionPeer.Team == null
-                || missionPeer.Team.Side == BattleSideEnum.None)
+                || crpgPeer.LastSpawnInfo == null)
```

Comment:
I think this would crash the server for any peer that has not the cRPGPeer set.
do
crpgPeer?.LastSpawnInfo == null
instead.

Also consider setting lastSpawnInfo == null on joining spectator.
That way people using spectator to suicide would not get rewarded (reward server checks last spawn info)